src/hooks/useNotifications.tsx

```diff
@@ -82,6 +84,7 @@ const useNotificationsContext = () => {
         [NotificationType.AbacusGenerated]: true,
         [NotificationType.SquidTransfer]: true,
         [NotificationType.ReleaseUpdates]: true,
+        [NotificationType.ChaosLabs]: true,
```

Comment:
This `NotificationType.ChaosLabs` should probably be bucketed w/ `ReleaseUpdates` or another `NotificationType`. Check out `PreferencesDialog` to see how these NotificationTypes are being used.

Reply:
good call 👍